malariagen_data/af1.py

```diff
@@ -91,6 +92,18 @@ def __init__(
         tqdm_class=None,
         **storage_options,  # used by fsspec via init_filesystem()
     ):
+        # If show_progress has not been specified, then determine the default.
+        if show_progress is None:
+            # Get the env var, if it exists.
+            show_progress_env = os.getenv("MGEN_SHOW_PROGRESS")
+
+            # If the env var does not exist, then use the class default.
+            # Otherwise, convert the env var value to a boolean and use that.
+            if show_progress_env is None:
+                show_progress = True
+            else:
+                show_progress = show_progress_env.lower() in ("true", "1", "yes", "on")
+
```

Comment:
The logic looks good but could move this up to `AnophelesBase.__init__`.

Reply:
Confusingly, `AnophelesBase.__init__` currently defaults `show_progress` to `False`, whereas `Ag3` and `Af1` currently default to `True`. I expect this is just an historical artefact, so I'll set the default as `True`.